Upstream: under review, https://gitlab.freedesktop.org/mesa/drm/-/merge_requests/109

From db66b10d790562b7247ea86adb497af9b7385cf6 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Fri, 15 Jan 2021 21:11:41 +0100
Subject: [PATCH] meson: Also search for rst2man.py

That's what upstream docutils installs by default.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index a4d8d707..07d2e086 100644
--- a/meson.build
+++ b/meson.build
@@ -261,7 +261,7 @@ else
 endif
 
 with_man_pages = get_option('man-pages')
-prog_rst2man = find_program('rst2man', required: with_man_pages == 'true')
+prog_rst2man = find_program('rst2man', 'rst2man.py', required: with_man_pages == 'true')
 with_man_pages = with_man_pages != 'false' and prog_rst2man.found()
 
 config.set10('HAVE_VISIBILITY',
-- 
2.30.0

